tcms/xmlrpc/sqls.py

```diff
@@ -1,15 +1,15 @@
 TP_CLEAR_ENV_GROUP = '''
```

Comment:
This file is covered by #159, no need to duplicate work. Also #159 is adding tests which cover the usage of these particular queries.